go/libraries/doltcore/sqle/statsnoms/iter.go

```diff
@@ -159,7 +159,11 @@ func (s *statsIter) Next(ctx *sql.Context) (sql.Row, error) {
 
 func (s *statsIter) ParseRow(rowStr string) (sql.Row, error) {
 	var row sql.Row
-	for i, v := range strings.Split(rowStr, ",") {
+	parts := strings.Split(rowStr, ",")
+	if len(parts) > len(s.currentTypes) {
```

Comment:
So I see how this prevents a panic. But is it correct? I imagine we want to parse as much as we can and return 0 values for the remaining columns, right?

Reply:
Ah i meant to revert, I was using to before to see if the database had any other loading issues. Thanks for pointing out